app/models/govuk_publishing_components/audit_applications.rb

```diff
@@ -76,8 +78,14 @@ def find_components(files, find, type)
       files.each do |file|
         src = File.read(file)
         components_found << find_match(find, src, type)
-        gem_references = find_gem_references(file, src)
-        @gem_style_references << gem_references if gem_references
+
+        if type == "javascripts"
+          jquery_references = find_code_references(file, src, /\$\(/)
```

Comment:
Should we include `jQuery()` as well?

Another thought is to look for the ESLint global declaration that stops it complaining about `$` being an undeclared variable:
```javascript
/* global $ */
```

Reply:
Hmm, having thought about this I can't find any file that exclusively uses `jQuery()` - so I don't think that this is relevant.